lib/Backend/IR.h

```diff
@@ -26,15 +26,29 @@ struct CapturedValues
     SListBase<ConstantStackSymValue> constantValues;           // Captured constant values during glob opt
     SListBase<CopyPropSyms> copyPropSyms;                      // Captured copy prop values during glob opt
     BVSparse<JitArenaAllocator> * argObjSyms;                  // Captured arg object symbols during glob opt
+    uint refCount;
 
-
-    CapturedValues() : argObjSyms(nullptr) {}
+    CapturedValues() : argObjSyms(nullptr), refCount(0) {}
     ~CapturedValues()
     {
         // Reset SListBase to be exception safe. Captured values are from GlobOpt->func->alloc
         // in normal case the 2 SListBase are empty so no Clear needed, also no need to Clear in exception case
         constantValues.Reset();
         copyPropSyms.Reset();
+        argObjSyms = nullptr;
+        refCount = 0;
```

Comment:
>refCount = 0; [](start = 8, length = 13)

Should there be an assert that its already 0? or may be <=1 . Anything more than 1 would mean someone is still pointing to this?

Reply:
Makes sense, I'll add it.